### DISCUSSION TITLE
App Showcase

### DISCUSSION THREAD

**Discussion by yeppiidev** [raspiduino/mre-sdk#6] [Ideas]
Send the applications you've made with the help of the documentation and resources mentioned here!

**Comment by raspiduino** [comment]
[XimikBoda](https://github.com/XimikBoda) and I made a [Linux emulator running on Nokia](https://github.com/raspiduino/uARMRE)

**Reply by yeppiidev** [reply]
https://discord.gg/b3pZ55XdXh I've made a discord in case anyone wants to send a lot of screenshots and videos
